cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -920,8 +920,8 @@ New Cypher command for administering privilege for listing constraints.
 | `CYPHER runtime=compiled` (Compiled runtime) | Functionality | Removed| Replaced by the new `pipelined` runtime which covers a much wider range of queries.
 | `CREATE INDEX [name] FOR (n:Label) ON (n.prop)` | Syntax | Added | New syntax for creating indexes, which can include a name.
 | `CREATE CONSTRAINT [name] ON ...` | Syntax | Extended | The create constraint syntax can now include a name.
-| `DROP INDEX name` | Syntax | Added | <<query-plan-drop-index-by-name,New command>> for dropping an index by name.
-| `DROP CONSTRAINT name` | Syntax | Added | <<query-plan-drop-constraint-by-name,New command>> for dropping a constraint by name, no matter the type.
+| `DROP INDEX name` | Syntax | Added | <<administration-indexes-drop-an-index,New command>> for dropping an index by name.
+| `DROP CONSTRAINT name` | Syntax | Added | <<administration-constraints-drop-constraint,New command>> for dropping a constraint by name, no matter the type.
```

Comment:
The constraint link does not seem to work when I build it locally, the index on does though.

Reply:
```suggestion
| `DROP CONSTRAINT name` | Syntax | Added | <<administration-constraints-syntax-drop,New command>> for dropping a constraint by name, no matter the type.
```

Reply:
Yup this is the new id I need after David made his changes on 4.3